eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -37,7 +37,8 @@ RestrictAddressFamilies=AF_UNIX
 RestrictRealtime=yes
 SystemCallErrorNumber=EPERM
 # @network-io is required for logging to the journal to work
-SystemCallFilter=~@resources @obsolete @mount @module @keyring @chown @privileged
+# @privileged is required for certain ostree operations
+SystemCallFilter=~@resources @obsolete @mount @module @keyring @chown
```

Comment:
The list I came up with for Mogwai also blacklisted `@clock @cpu-emulation @debug @raw-io`. Can you blacklist any of those in addition?

Nitpick: It would also be a little easier to compare this list to the man page if the categories were in alphabetical order.

Reply:
Oh, also, from reading `man systemd.exec` in more detail, I learned that you also need to add:
```
SystemCallArchitectures=native
```
to prevent programs bypassing the blacklist by running in x86 mode on an x86_64 machine (for example).

Reply:
See: https://github.com/endlessm/mogwai/blob/master/mogwai-scheduled/mogwai-scheduled.service.in